### ISSUE TITLE
Add Obsidian to Docs Home Page

### ISSUE THREAD

**Issue by mason-at-pieces** [pieces-app/documentation#54]
Need to add the Obsidian logo to the products list on the docs home page.

<img width="783" alt="Screenshot 2023-06-07 at 12 50 19 AM" src="https://github.com/pieces-app/documentation/assets/123421085/ad7b7fa0-8e2a-4da0-b22f-48c0bc4a6f6d">

The more ideal solution here would be to have this list generated by a centralized list of our products so we don't have to keep updating this. This goes for the version list in `docs.json` too

**Comment by jordan-pieces** [comment]
Yeah that is the foal here @mason-at-pieces and is related to the version numbers too. I was thinking about doing this via github actions, but it has fallen on the back-burner due to other priorities, do you have thoughts here? Feel free to share them below would love to talk more about this before implementation - but should have time for this soon. 

At minimum I can add these as variables in docs.json so we can at least have the data centralized on the docs.
